### PR TITLE
Add some missing translatable strings

### PR DESCRIPTION
**Pull request by pvagner**
Came accross a few missing translatable strings.

### CODE REVIEW

File: src/components/views/room_settings/AliasSettings.js [hunk 1]
```diff
@@ -253,7 +253,7 @@ module.exports = React.createClass({
 
         return (
             <div>
-                <h3>Addresses</h3>
+                <h3>{ _t('Addresses') }</h3>
                 <div className="mx_RoomSettings_aliasLabel">
                     { _t('The main address for this room is') }: { canonical_alias_section }
```
Comment (t3chguy) [inline src/components/views/room_settings/AliasSettings.js]: OOI should the above end up as a parameter of the translation rather than this concatenation?

**Comment by pvagner** [comment]
Oh yeah, so it should also be added to english? I'm sorry I assumed this will be generated automagically.

**Comment by t3chguy** [comment]
I think there's a script to run, which will generate it automatically, but its a development task

**Comment by t3chguy** [comment]
https://github.com/matrix-org/matrix-react-sdk/blob/develop/scripts/gen-i18n.js

**Comment by lukebarnard1** [comment]
Hmm, looks like this conflicts with my earlier changes to PowerSelector. 

@pvagner if you feel like fixing the conflicts, that'd be great, although I suspect I may have fixed vector-im/riot-web#5547 inadvertently, so it might be worth checking if that's still an issue on `/develop`.

**Comment by pafcu** [comment]
I also broke it by removing `_tJsx()` :-(

You can use `_t()` instead now. Let me know if you need help with making the changes.

**Comment by pvagner** [comment]
@lukebarnard1 My change related to the PowerSelector is no longer needed as you have already addressed that.
I have to revert it and add some more unlabeled stuff e.g. not translatable alt texts on the homepage.

**Comment by lukebarnard1** [comment]
@pvagner let me know when you need me to take another look at this.

**Comment by pvagner** [comment]
@lukebarnard1: Well, I think this is done. I consider this as a minor thing. The only thing I don't know how to handle is to satisfy @t3chguy's review request.
Everything is committed and merge conflicts are solved.
If I were you I would just accept the PR. But still if you find something you think I might have overlooked, then I'm happy to correct it right away.

**Comment by lukebarnard1** [comment]
Okay, I shall merge because LGTM. Thanks!

**Pull request merged**